server/routes/bio.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Commons Biology Explorer routes
+"""
+
+from flask import Blueprint, render_template
+import routes.api.shared as shared_api
+
+bp = Blueprint('bio', __name__, url_prefix='/bio')
+
+
+@bp.route('/')
+def main():
```

Comment:
nit: restrict these to non-staging/prod

Reply:
Done

Reply:
There will be more bio-related pages including a slightly modified version of the place browser. The Disease and Protein browser will contain summary graphs representing the data related to each of these entitites.